ui/common/src/coordsColour.ts

```diff
@@ -8,8 +8,25 @@ export default function changeColourHandle(){
 		"blue3": ["#d9e0e6", "#315991"],
 		"canvas": ["#d7daeb", "#547388"],
 		"wood": ["#d8a45b", "#9b4d0f"],
+		"wood2": ["#a38b5d", "#6c5017"],
+		"wood3": ["#d0ceca", "#755839"],
+		"maple": ["#e8ceab", "#bc7944"],
+		"maple2": ["#E2C89F", "#996633"],
+		"leather": ["#d1d1c9", "#c28e16"],
+		"green": ["#FFFFDD", "#6d8753"],
 		"brown": ["#F0D9B5", "#946f51"],
-		"pink": ["#f1f1c9", "#f07272"]
+		"pink": ["#E8E9B7", "#ED7272"],
+		"marble": ["#93ab91", "#4f644e"],
+		"blue-marble": ["#EAE6DD", "#7C7F87"],
+		"green-plastic": ["#f2f9bb", "#59935d"],
+		"green-glass": ["#fafa97", "#e4a836"],
```

Comment:
This is great!
I suggest changing line 22 from
"green-glass": ["#fafa97", "#e4a836"],
to
"wood4": ["#caaf7d","#7b5330"],
as the theme green-glass does not exist anymore, and wood4 is missing.